sub-packages/bionemo-core/tests/bionemo/core/data/test_load.py

```diff
@@ -45,8 +45,9 @@ def test_load_raises_error_on_invalid_tag(tmp_path):
 def test_load_cli():
     # It looks like there's some issues with our NGC resources, but this is blocking CI. TODO: Revert to ngc when these
     # resources are available.
+    # FIXME (dorotat): set source=ngc once the access issue with NGC is resolved (https://github.com/NVIDIA/bionemo-framework/issues/682)
     result = subprocess.run(
-        ["download_bionemo_data", "--source", "ngc", "single_cell/testdata-20240506"],
+        ["download_bionemo_data", "--source", "pbss", "single_cell/testdata-20240506"],
```

Comment:
Can we just xfail this test for now? This will never pass in our pre-merge CI that runs off-prem